### PR TITLE
feat: 🎸 add helper text option to checkbox component

### PR DESCRIPTION
**Pull request by randallmorey**
This PR adds support for helper text to the checkbox component, bringing it in line with other form components that already support help.  Helper text is optional.

Screenshot 2021-05-21 at 13 47 55 

Screenshot 2021-05-21 at 13 48 36
Screenshot 2021-05-21 at 13 48 30
Screenshot 2021-05-21 at 13 48 02

### CODE REVIEW

File: addons/rose/tests/integration/components/rose/form/checkbox-test.js [hunk 1]
```diff
@@ -30,6 +30,17 @@ module('Integration | Component | rose/form/checkbox', function (hooks) {
     assert.ok(await find('input'));
   });
 
+  test('it renders optional helper text', async function (assert) {
+    assert.expect(1);
+    await render(
+      hbs`<Rose::Form::Checkbox @label="Label" @helperText="Hello world" />`
+    );
+    assert.equal(
+      await find('.rose-form-helper-text').textContent.trim(),
```
Comment (gsusmi) [inline addons/rose/tests/integration/components/rose/form/checkbox-test.js:39]: Is `await` needed for `find`? Looking through rose test files, we seem to have a mix of tests using `find` as both synchronous and asynchronous actions.

Ref: https://github.com/emberjs/ember-test-helpers/blob/master/API.md#find

Comment (randallmorey) [reply addons/rose/tests/integration/components/rose/form/checkbox-test.js]: Not necessary.  It looks like a bunch of form component tests have that.  I removed for this one.

**Review by gsusmi — APPROVED** [review]
LGTM. Ty for catching this!

**Review by calcaide — APPROVED** [review]
LGTM

**Pull request merged**